submissions/TatyanaMolchanova/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,117 @@
+.nav-link img {
+    max-width: 30px;
+}
+.nav {
+    float: right;
+}
+.nav-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;  
+    width: 300px;
+    list-style-type: none;
+    padding-right: 30px;
+}
+.nav-item a {
+    text-decoration: none;
+    color: gray;
+}
+
+.popup-link img {
+    max-width: 50px;
+}
+
+#myhtmlpopup {
+    display: none;
+    position: absolute;
+    top: 40px;
+    width: 300px;
+    left: -200px;
+    max-height: 250px;
+    max-height: 256px;
+    overflow: auto;
+    box-shadow:1px -2px 10px 0px rgba(50, 50, 50, 0.5)
+}
+
+#mypopup:checked ~ #myhtmlpopup {
+    display: block;
+}
+
+#mypopup {
+    position: absolute;
+    top: 32%;
+    left: 34%;
+    transform: translate(-50%, -50%);
+    opacity: 0;
+    height: 50px;
+    width: 50px;
+}
+
+.popup {
+    padding: 15px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.popup-link {
+    display: flex;
+    flex-direction: column;
+}
+
+.popup-item {
+    width: 33%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+a, a:focus, a:hover, a:active {
+    text-decoration: none;
+}
+
+:focus, 
+#mypopup:checked ~ .nav-link-btn,
+.nav-item:hover, 
+.nav-item:active, 
+.popup-item:hover, 
+.popup-item:focus, 
+.popup-item:active {
+    outline: blue auto 1px;
+}
+
+.read-more-state {
+    display: none;
+  }
+  
+  .read-more-target {
+    display: none;
+    max-height: 0;
+    transition: .25s ease;
+  }
+  
+  .read-more-state:checked ~ .read-more-wrap .read-more-target {
+    display: block;
+    max-height: 999em;
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
+  .read-more-state ~ .read-more-trigger:before { 
```

Comment:
Screen reader and search bot can't read this text. Move it to the HTML.

Reply:
I did it.